packages/sheets-numfmt/src/controllers/numfmt.menu.controller.ts

```diff
@@ -50,4 +54,12 @@ export class NumfmtMenuController extends Disposable {
         this.disposeWithMe((this._componentManager.register(MORE_NUMFMT_TYPE_KEY, MoreNumfmtType)));
         this.disposeWithMe((this._componentManager.register(OPTIONS_KEY, Options)));
     }
+
+    public setCurrencySymbol(symbol: keyof typeof currencyMap) {
```

Comment:
JSDoc is expected to be required on this method if it is intended to be used directly by users.